### PR TITLE
Add Desktop vs Mobile post

### PR DESCRIPTION
**Pull request by mfwmyfacewhen**
I'd like an article I can link people to whenever this topic comes up.

### CODE REVIEW

**Comment by beerisgood** [comment]
> here is a pervasive idea that mobile devices can't be private or secure and that you should switch to a desktop machine
>

Mobile phones are much more secure and private than desktop systems.

**Comment by mfwmyfacewhen** [comment]
> > here is a pervasive idea that mobile devices can't be private or secure and that you should switch to a desktop machine
> 
> Mobile phones are much more secure and private than desktop systems.

Yeah that's my point. I wanted to make something to link people to when they say not to use phones etc

File: content/posts/knowledge/Desktop vs Mobile.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Desktop vs Mobile"
+date: 2023-01-27
+tags: ['Knowledge base', 'Privacy, 'Security']
+author: mfwmyfacewhen
+---
+
+There is a pervasive idea that mobile devices can't be private or secure and that you should switch to a desktop machine. This idea can harmful. Phones can be a valuable tool that can enhance many threat models.
```
Comment (mfwmyfacewhen) [inline content/posts/knowledge/Desktop vs Mobile.md:8]:
```suggestion
There is a pervasive idea that mobile devices can't be private or secure and that you should switch to a desktop machine. This idea can be harmful. Phones can be a valuable tool that can enhance many threat models.
```

**Comment by wj25czxj47bu6q** [comment]
Might be helpful:

- https://github.com/GrapheneOS/platform_build/commit/cc3a7256f89eccaf88ade7e3777aaf3f3b91b502
- https://source.android.com/docs/security (and its many subpages)
- https://madaidans-insecurities.github.io/android.html
- https://madaidans-insecurities.github.io/security-privacy-advice.html#operating-system
- https://madaidans-insecurities.github.io/guides/linux-hardening.html#verified-boot
- https://madaidans-insecurities.github.io/linux-phones.html

File: content/posts/knowledge/Desktop vs Mobile.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Desktop vs Mobile"
+date: 2023-01-27
+tags: ['Knowledge base', 'Privacy, 'Security']
```
Comment (noClaps) [inline content/posts/knowledge/Desktop vs Mobile.md:4]:
```suggestion
tags: ['Knowledge base', 'Privacy', 'Security']
```
I'm pretty sure this is why the build is failing

**Pull request closed without merging**